components/selectivity/selectivity-options-container.ts

```diff
@@ -231,16 +234,17 @@ export class SelectivityOptionsContainer {
       }
 
       this.options.selectivity.active.push(value);
-      this.options.selectivity.data.next(this.options.selectivity.active);
+      this.options.selectivity.dataChange.emit(this.options.selectivity.active);
       this.options.selectivity.doEvent('selected', value);
     }
 
     if (this.options.selectivity.multiple === false) {
       this.options.selectivity.active[0] = value;
-      this.options.selectivity.data.next(this.options.selectivity.active[0]);
+      this.options.selectivity.dataChange.emit(this.options.selectivity.active[0]);
       this.options.selectivity.doEvent('selected', value);
       // turn back focus to input from options
-      this.options.selectivity.element.nativeElement.children[1].children[0].focus();
+      // TODO this crashed because of 'children' undefined:
+      // this.options.selectivity.element.nativeElement.children[1].children[0].focus();
```

Comment:
@valorkin please take a look at this commented line. I just commented it out because I didn't have much time to figure out what it really does. But there's some undefined element in there.